docs/usage/configuration/config_documentation.md

```diff
@@ -3832,16 +3832,22 @@ Sub-options for this setting include:
 * `system_mxid_display_name`: set the display name of the "notices" user
 * `system_mxid_avatar_url`: set the avatar for the "notices" user
 * `room_name`: set the room name of the server notices room
+* `room_avatar_url`: set the room avatar of the server notices room _Added in Synapse 1.98.0._
+* `room_topic`: set the room topic of the server notices room _Added in Synapse 1.98.0._
 * `auto_join`: boolean. If true, the user will be automatically joined to the room instead of being invited.
   Defaults to false. _Added in Synapse 1.98.0._
 
+Note that it will only be updated when a new notice event is sent.
+
 Example configuration:
 ```yaml
 server_notices:
   system_mxid_localpart: notices
   system_mxid_display_name: "Server Notices"
   system_mxid_avatar_url: "mxc://server.com/oumMVlgDnLYFaPVkExemNVVZ"
   room_name: "Server Notices"
+  room_avatar_url: "mxc://server.com/oumMVlgDnLYFaPVkExemNVVZ"
```

Comment:
```suggestion
  room_avatar_url: "mxc://example.com/oumMVlgDnLYFaPVkExemNVVZ"
```

since `server.com` is apparently a thing:

```
server.com.		287	IN	A	52.8.126.80
```